.github/workflows/ci.yml

```
@@ -0,0 +1,59 @@
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report
+          path: coverage/
+
+      # Setup QEMU (If Docker is required)
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+
+      # Setup Docker Buildx (If Docker is required)
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)